sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/UnaryPrefixOperatorRepeated.cs

```diff
@@ -69,7 +70,7 @@ protected override void Initialize(SonarAnalysisContext context)
                     }
 
                     var errorLocation = new TextSpan(topLevelUnary.SpanStart, lastUnary.OperatorToken.Span.End - topLevelUnary.SpanStart);

```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png 'Severity: BLOCKER') 'topLevelUnary' is null on at least one execution path. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS2259)